src/components/Perils/data/usePerils.ts

```diff
@@ -0,0 +1,54 @@
+import axios from 'axios'
+import { useEffect, useState } from 'react'
+import { Peril, TypeOfContract } from '../types'
+
+export const usePerils = (insuranceType: TypeOfContract) => {
+  const [perils, setPerils] = useState<[] | Peril[]>([])
+
+  const fetchPerils = async () => {
+    const url = `https://graphql.dev.hedvigit.com/graphql`
```

Comment:
I just noticed but i guess this should be the same as https://github.com/HedvigInsurance/web-next/pull/401/files#diff-01b4c9453b6be627da5698a4789ff9c3R131 - maybe we should put the host in a common place?